Classes/Emogrifier.php

```diff
@@ -430,7 +428,11 @@ protected function process(\DOMDocument $xmlDocument)
             $this->removeInvisibleNodes($xPath);
         }
 
+        $this->removeImportantAnnotationFromAllInlineStyles($xPath);
+
         $this->copyCssWithMediaToStyleNode($xmlDocument, $xPath, $cssParts['media']);
```

Comment:
Could you please check whether we could remove some blank lines from this method so the code lines are grouped together (so that related operations stand together)?